download_data.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
```

Comment:
Is it possible to add some descriptions for these samples or have a file described what these samples are.

Reply:
yeah, I think that is a good idea. I will add a brief description here, but a more thorough description should exist as some sort of table, text, or FAQ in the README or github wiki